cmd/oci/oci.go

```diff
@@ -60,13 +61,30 @@ var OCICmd = &cobra.Command{
 			os.Exit(1)
 		}
 		platform = p
+		nameMap := make(map[string]string)
+		originalName := artifact.Name
+		var newName string
+
+		if tag != "" {
```

Comment:
If a user did not pass the `tag` flag and `artifact.Name` doesn't include a `:`, we should append `:latest` tag.